img/media-types.dot

```diff
@@ -14,9 +13,5 @@ digraph G {
   imageIndex -> manifest [label="1..*"]
   manifest -> config [label="1..1"]
   manifest -> layer [label="1..*"]
-  artifact -> manifest [label="0..1"] [constraint = false];
-  artifact -> artifact [label="0..1"];
   manifest -> manifest [label="0..1"];
-  artifact -> imageIndex [label="0..1"]
-  artifact -> layer[label="0..*"]
```

Comment:
Will 0-layer images be supported?

Reply:
Aren't 0-layer images already supported today?

```json
$ crane manifest tianon/scratch:oci
{
  "schemaVersion": 2,
  "mediaType": "application/vnd.oci.image.manifest.v1+json",
  "config": {
    "mediaType": "application/vnd.oci.image.config.v1+json",
    "size": 290,
    "digest": "sha256:132144ab4e026a9b1ca95b5dfeb9892d12f0c2c1368ae0b4bdca408e62768396"
  },
  "layers": []
}
```

(Maybe the spec disagrees, but as an admittedly non-image-spec maintainer I don't see a good reason for it to do so :see_no_evil:)

Reply:
Currently I feel the spec can be interpreted as Layer[0] is required. (MUST/IS) we should update this to call out that Layers are option and if used to define an  image ... and so forth.

Reply:
I feel like this has been unclear and therefore exists in both ways. If we firm up the language to explicitly allow zero-layer images, then it should _not_ be considered a breaking change.